src/retrace/retrace.py

```diff
@@ -378,7 +378,7 @@ def run_gdb(savedir, plugin):
 
         batfile = os.path.join(savedir, "gdb.sh")
         with open(batfile, "w") as gdbfile:
-            gdbfile.write("%s -batch " % plugin.gdb_executable)
+            gdbfile.write("#!/usr/bin/sh\n\n%s -batch " % plugin.gdb_executable)
```

Comment:
Why?

Reply:
Without the shebang, `podman run` fails with an "exec format error".

Reply:
Ohhhhh, right. I got confused, thinking that it was GDB executing the script. Makes sense.